mne/forward/_field_interpolation.py

```diff
@@ -331,6 +332,8 @@ def make_field_map(evoked, trans='auto', subject=None, subjects_dir=None,
         Either `'accurate'` or `'fast'`, determines the quality of the
         Legendre polynomial expansion used. `'fast'` should be sufficient
         for most applications.
+    head_surf : bool
+        Get the subject head surface if True.
```

Comment:
...and I guess it should be `meg_surf`, because the `eeg_surf` is always the head